src/lib.rs

```diff
@@ -494,7 +682,7 @@ mod test {
 
     #[test]
     fn test_convert_from_bools() {
```

Comment:
I'm seeing compilation errors because `Vec` and `vec!` are not defined in tests. Please add the relevant imports so that the tests run in all 3 cases (stable, nightly, nightly+unstable).